packages/react-instantsearch/src/connectors/connectInfiniteHits.js

```diff
@@ -5,6 +5,10 @@ import {PropTypes} from 'react';
 
 import createConnector from '../core/createConnector';
 
+function getId() {
+  return 'pager';
```

Comment:
Why pager instead of p? Do you want to handle the case where the user might use infiniteHits along with a normal pagination widget?